### DISCUSSION TITLE
CORS of Vercel sites

### DISCUSSION THREAD

**Discussion by horsegame** [vercel/community#3411] [Help]
### Summary

Need some help to add CORS headers to Vercel 308 redirects. See #1939 .

### Steps to Reproduce

Follow https://vercel.com/guides/how-to-enable-cors#enabling-cors-using-vercel.json to setup vercel.json and send GET requests to urls with double slashes. It will trigger a 308 redirect without CORS headers.

**Comment by deanshub** [comment]
it happened to me because I had my domain redirect to `www`
I had both `my.domain` and `www.my.domain` configured in vercel but 
`my.domain` was redirected to `www.my.domain`
after doing it the other way around
`www.my.domain` redirected to `my.domain`
all was working well

see https://stackoverflow.com/questions/75062050/stripe-webhook-returning-308-error-when-calling-vercel-serverless-function
